source/ContentScript/index.ts

```diff
@@ -7,11 +7,14 @@ import {browser} from 'webextension-polyfill-ts';
 const controller = SpellcheckController.getInstance();
 
 let userFilters: Filter[] = [];
-// whether the plugin is active
-let active = false;
+// whether the plugin is active on the current page
+let activeOnPage = true;
```

Comment:
Why make a distinction between those two?

Reply:
This variable is used in other places with the 'active' from storage. I thought it was a good idea to make it clear that this variable applies only to the current page.

Reply:
I feel a "page active" and a "plugin active" is confusing for the user. If you can revert that I'll merge this.

Reply:
You mean the naming in the code or the entire behavior of per-page control?

Reply:
I mean the entire behaviour of the page control